src/docs/quick-start.rst

```diff
@@ -1,12 +1,201 @@
+===========
 Quick Start
 ===========
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec ac ligula et felis aliquam posuere. Etiam ac augue vel tortor porttitor placerat vitae ac libero. Nunc a efficitur lectus. Aliquam malesuada congue arcu quis luctus. Quisque pretium dolor orci, ut ullamcorper felis tincidunt eget. Duis auctor felis ut maximus facilisis. Nam eu feugiat tortor.
+This page provides brief instructions to configure SSSD with FreeIPA, AD, and LDAP.
+
+Quick Start IPA
+***************
+
+Before starting, make sure you have the following information.
+
+- Administrator credentials *i.e. admin*
+- FreeIPA domain name *i.e. sssd.io*
+- FreeIPA server hostname *i.e. ipa1.sssd.io*
+- FreeIPA server IP *i.e. 1.2.3.4*
+
+Install the necessary packages, for RHEL and clones the package is named ``ipa-client``, and for Fedora it’s ``freeipa-client``.
+
+.. code-tabs::
+
+    .. fedora-tab::
+
+         dnf install -y freeipa-client
+
+    .. rhel-tab::
+
+        yum install -y ipa-client
+
+Make sure DNS is pointing to your IPA server(s) by checking ``/etc/resolv.conf`` and it having it contain an entry with the IPA server IP, if it’s missing go ahead and add it, as the first entry.
```

Comment:
This may be a bit tricky as `/etc/resolv.conf` can be overwritten by `NetworkManager` once a while. I think it is worth mentioning that.

Reply:
There are number of other things that can overwrite it, e.g. resolved. We can't cover them all so it is better to avoid this area.